projects/plugins/jetpack/tests/php/sync/server/class.jetpack-sync-test-replicastore.php

```diff
@@ -172,7 +173,7 @@ public function get_comment( $id ) {
 		return false;
 	}
 
-	public function upsert_comment( $comment ) {
+	public function upsert_comment( $comment, $silent = false ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
```

Comment:
I only see one call with this second param, which doesn't do anything. Would it be better to just remove it from the call vs. adding it as an unused param?

(On that note, `upsert_post` also has the same unused param.)

Reply:
OTOH, it matches the non-test class at fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Szh%2Qcyhtvaf%2Swrgcnpx%2Sflap%2Spynff.wrgcnpx%2Qflap%2Qfunqbj%2Qercyvpnfgber.cuc%3Se%3Qop1q344r%233119-og now.

Reply:
Good call. I tried to be thorough but didn't spot that function. So definitely leave it how you had it. :^)